README.md

```diff
@@ -1,68 +1,109 @@
+```shell
+$ npm install
+```
 
-## Learn More
+to download and install the dependencies.
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+### Firebase
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+---
 
-### Code Splitting
+A Firebase project must be created to fully test and run the app. You can sign up using your google account here:
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/code-splitting
+https://firebase.google.com/
 
-### Analyzing the Bundle Size
+Create a project and call it `campus-gaming-network-test` or something along those lines. You will need to initialize a new database as well as enable the `Email/Password` authentication method. You can view the Firebase documentation here:
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size
+https://firebase.google.com/docs
 
-### Making a Progressive Web App
+To register the app to your firebase account you must:
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app
+1. Copy the contents of `.env.sample` into a new file called `.env.local`
 
-### Advanced Configuration
+2. Populate the firebase section of that `.env.local` file with your unique Firebase SDK snippet found under your firebase project settings.
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
+```javascript
+const firebaseConfig = {
+  apiKey: 'AIzaSyDOCAbC123dEf456GhI789jKl01-MnO',
+  authDomain: 'myapp-project-123.firebaseapp.com',
+  databaseURL: 'https://myapp-project-123.firebaseio.com',
+  projectId: 'myapp-project-123',
+  storageBucket: 'myapp-project-123.appspot.com',
+  messagingSenderId: '65211879809',
+  appId: '1:65211879909:web:3ae38ef1cdcb2e01fe5f0c',
+};
+```
 
-### Deployment
+```
+REACT_APP_FIREBASE_API_KEY=xxxxXXXX
+REACT_APP_FIREBASE_AUTH_DOMAIN=xxxxXXXX.firebaseapp.com
+REACT_APP_FIREBASE_DATABASE_URL=https://xxxXXXX.firebaseio.com
+REACT_APP_FIREBASE_PROJECT_ID=xxxxXXXX
+REACT_APP_FIREBASE_STORAGE_BUCKET=xxxxXXXX.appspot.com
+REACT_APP_FIREBASE_MESSAGING_SENDER_ID=xxxxXXXX
+REACT_APP_FIREBASE_APP_ID=xxxxXXXX:xxxxXXXX:xxxxXXXX:xxxxXXXX
+```
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
+3. To upload the cloud functions into you firebase project run,
 
-### `npm run build` fails to minify
+```shell
+$ firebase deploy
+```
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+4. To upload the schools info into the db, navigate to the `uploadSchools` folder in the app directory and run,
```

Comment:
`npm install` just installs the project dependencies but doesn't actually run the script, we should add instructions on how to run it like `node .` and what you can expect

Reply:
Copy that!